src/irmin-test/store.ml

```diff
@@ -17,6 +17,8 @@
 open Lwt.Infix
 open Common
 
+let ( let* ) x f = Lwt.bind x f
```

Comment:
unfortunately this doesn't work on 4.07 :p

Reply:
Ah, indeed. I was getting ahead of myself since we're on 4.08 in `irmin-pack` now :slightly_smiling_face: